src/utils/constants.ts

```diff
@@ -1 +1,2 @@
 export const MUTEZ_FACTOR = 1000000;
+export const WALLET_PRIVATE_KEY_FILE = `payout_wallet_private.key`;
```

Comment:
add a note in the README to show users an example / template?

Reply:
We have to add both - the key file and config.hjson - template actually.